src/main/scala/akka/minion/App.scala

```diff
@@ -59,12 +57,14 @@ object App {
         apiCallPerHour = config.getInt("max-api-calls-per-hour"),
         teamMembers = config.getStringList("team-members").asScala.toSet,
         bots = config.getStringList("bots").asScala.toSet,
-        teamRepos = teamRepos
+        teamRepos = teamRepos,
+        statsRepos = config.getStringList("stats-repos").asScala.toList
       )
 
       system.actorOf(App.props(settings), "minion-supervisor")
-
       Await.result(system.whenTerminated, Duration.Inf)
+
+//      new Graphql(settings).call()
```

Comment:
Ah, so this is for now only called manually and not actually visible on any of the minion pages?

Maybe put it in a separate `Main`?

Reply:
Yes, it can be improved. 
Would be great to rewrite the data fetching for the Minion pages with Graphql, chances are it gets faster.